app/controllers/api/v1/users_controller.rb

```diff
@@ -5,11 +5,10 @@ module V1
     # API for application users
     class UsersController < Api::V1::ApiController
       before_action :set_user, only: %i[show]
-      before_action :authorize_user, only: %i[show]
+      before_action :set_users, only: %i[index]
 
       # GET /users
       def index
-        @users = policy_scope(User)
         authorize User
```

Comment:
Several tests fail after removing this authorize method, so it might be better to keep it rather than relying on just the set_users

Reply:
See my comment below - I think we have some conflicting behavior in the `ApplicationPolicy` that every policy inherits from and the `UserPolicy` - we might have to explicitly call authorize in `index` in order to get the `UserPolicy.index?` method to be triggered.  Would love to think about that and maybe specify this behavior in all other models as well?  We should chat about this, maybe later today or next week.